assignment/api/src/resources/assignment_es.properties

```diff
@@ -1054,7 +1054,9 @@ sendFeedback.error=Error al devolver el feedback a {0} estudiantes.
 # Peer Review
 peerAssessmentSavedGrading=Comentarios y/o nota han sido guardados pero no entregados. Si no entrega su revisi\u00f3n antes de la fecha establecida, ser\u00e1 entregada por usted.
 peerAssessmentSavedSubmission=Los comentarios y/o la nota han sido guardados y entregados.
-peerAssessmentName=Evaluaci\u00f3n por pares - Los estudiantes se eval\u00faan entre ellos
+peerAssessmentName=Evaluaci\u00f3n por pares - {0}
+peerAssessmentStudents=Los estudiantes se eval\u00faan entre ellos
```

Comment:
Please, put all the new translations at the end of the file with the S2U-34 prefix